src/styles/partials/components/_accordion.scss

```diff
@@ -0,0 +1,74 @@
+.dg_accordion__collapsible {
+  color: $blue;
+  font-family: $default-heading-font;
+  font-size: $h2-font-size;
+  font-weight: $font-bold;
+  width: 100%;
+  border-bottom: $border-width-small $black solid;
+  &:hover {
+    color: $vibrant-blue;
+  }
+  .dg_accordion__subheader {
+    font-size: $smallest-heading-font-size;
+    font-weight: $font-medium;
+    color: $gray;
+    padding-top: $padding-small;
+    text-transform: none;
+  }
+  .dg_accordion-item-body {
+    text-align: left;
+    font-size: $smallest-heading-font-size;
+    font-weight: $font-medium;
+    padding: $default-padding;
+    letter-spacing: 0;
+    color: $gray;
+    text-transform: none;
+  }
+  .dg_accordion-btn {
+    position: relative;
+    padding-top: $padding-large;
+    padding-bottom: $padding-small;
+    width: 100%;
+    text-align: left;
+    text-transform: uppercase;
+    height: auto;
+    font-family: $default-heading-font;
+    font-size: $small-heading-font-size;
+    font-weight: $font-bold;
+    cursor: pointer;
+    &:focus {
+      color: $blue;
+      outline: none;
+    }
+    &:after {
+      position: absolute;
+      padding-top: $padding-small;
+      padding-bottom: $padding-small;
+      font-size: $h1-font-size;
+      @include icon-content("\f107"); /*Font awesome right arrow*/
+      top: 0;
```

Comment:
based on the icon change, i believe this needs to be 8px. We will have to confirm with design.

Reply:
@tmccoy529 still missing this one.